README.md

```diff
@@ -198,11 +197,11 @@ You can hook up any key with one of several actions:
   exec = "some_command --with-parameters"
 ```
 
-#### Emulate a key-press
+#### Emulate a (series of) key-press
 
 ```
 [keys.action]
-  keycode = "114"
+  keycode = "9-20+200 / 46"
```

Comment:
I feel like we should have two examples here, actually: one with a simple keycode, the other explaining serial key-codes and the delays between them.